src/main/java/com/parasoft/demoapp/config/rabbitmq/RabbitMQInventoryResponseQueueListener.java

```diff
@@ -0,0 +1,48 @@
+package com.parasoft.demoapp.config.rabbitmq;
+
+import com.parasoft.demoapp.dto.InventoryOperationRequestMessageDTO;
+import com.parasoft.demoapp.dto.InventoryOperationResultMessageDTO;
+import com.parasoft.demoapp.service.OrderMQService;
+import com.parasoft.demoapp.service.OrderService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.amqp.core.Message;
+import org.springframework.amqp.rabbit.config.SimpleRabbitListenerContainerFactory;
+import org.springframework.amqp.support.converter.MessageConverter;
+import org.springframework.stereotype.Component;
+
+@Slf4j
+@Component
+public class RabbitMQInventoryResponseQueueListener extends RabbitMQRefreshableMessageListener {
+
+    private final MessageConverter rabbitMqMessageConverter;
+    private final OrderService orderService;
+    private final OrderMQService orderMQService;
+
+    public RabbitMQInventoryResponseQueueListener(MessageConverter rabbitMqMessageConverter,
+                                                  OrderService orderService,
+                                                  OrderMQService orderMQService,
+                                                  SimpleRabbitListenerContainerFactory simpleRabbitListenerContainerFactory) {
+        super(simpleRabbitListenerContainerFactory,
+                RabbitMQConfig.DEFAULT_ORDER_SERVICE_RESPONSE_QUEUE);
```

Comment:
We can't use DEFAULT_ORDER_SERVICE_RESPONSE_QUEUE since the listened queue may be changed(on demo admin page) before PDA re-starts up. Use RabbitMQConfig.getOrderServiceListenToQueue() method.